### DISCUSSION TITLE
How to delete https://username.github.io page?

### DISCUSSION THREAD

**Discussion by IOLOII** [community/community#52241] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Now I had publish a github page : https://ioloii.github.io/IOLOII/
But I found that already have a github page :  https://ioloii.github.io
I don't know and couldn't find where publish this page (https://ioloii.github.io)
I just want to delete this page : https://ioloii.github.io and want to change url( https://ioloii.github.io/IOLOII/) to this url (https://ioloii.github.io) which not have suffix

Help.
![image](https://user-images.githubusercontent.com/34856171/230573315-0a148a64-731a-41b6-8dfd-f346472e34e7.png)

**Comment by BREN0sx** [comment]
Hey! For that, you will have to rename or create a repository with the name `ioloii.github.io`, simple as that. This way the index will be reflected in the main url, without using the suffix

Hope this helps! 😉

**Comment by IOLOII** [comment, marked as answer]
Yes got this，Thank you
from 阿里邮箱 iPhone------------------------------------------------------------------
***@***.***>
日　期：2023年04月07日 21:31:18
***@***.***>
***@***.***>; ***@***.***>
主　题：Re: [community/community] How to delete https://username.github.io page? (Discussion #52241)

Hey! For that, you will have to rename or create a repository with the name ioloii.github.io, simple as that. This way the index will be reflected in the main url, without using the suffix
Hope this helps! 😉

**Reply by BREN0sx** [reply]
Right! Excellent 👊

**Comment by brianmapes** [comment]
when I try this instruction, it says "username.github.io already exists in this account". How can I find which out which of my many repos owns the Hello World page at username.github.io, so I can delete it? I made it many years ago I think.

**Comment by EdwinFlores19** [comment]
Hey @IOLOII,

To get the content from your 

https://ioloii.github.io/IOLOII/ page to show up on your main https://ioloii.github.io URL, you need to do two things:

Find the repository that creates your main page (

https://ioloii.github.io), which you couldn't find before. By GitHub's convention, it must be named exactly 

ioloii.github.io. You need to delete or rename this repository first.

Rename your IOLOII repository to ioloii.github.io.

That's it! GitHub will automatically publish it to the main URL.


Spanish version:

Hola @IOLOII,

Para que el contenido de tu página 

https://ioloii.github.io/IOLOII/ aparezca en tu URL principal https://ioloii.github.io, necesitas hacer dos cosas:

Encuentra el repositorio que crea tu página principal (

https://ioloii.github.io), el cual no podías encontrar antes. Por convención de GitHub, debe llamarse exactamente 

ioloii.github.io. Primero necesitas eliminar o renombrar este repositorio.

Renombra tu repositorio IOLOII a ioloii.github.io.

¡Eso es todo! GitHub lo publicará automáticamente en la URL principal.
